package.json

```diff
@@ -9,7 +9,7 @@
     "type": "git",
     "url": "https://github.com/JedWatson/react-tappable.git"
   },
-  "dependencies": {
+  "peerDependencies": {
```

Comment:
Also needs to be a devDependency if we make this change, or the examples fail to build / run

See https://github.com/JedWatson/react-select/blob/master/package.json for an example